src/Rules.py

```diff
@@ -84,7 +85,15 @@ def checkRequireStringForArea(state: CollectionState, area: dict):
             func_args = item[1].split(",")
             if func_args == ['']:
                 func_args.pop()
-            func = getattr(Rules, func_name)
+
+            func = globals().get(func_name)
+
+            if func is None:
+                func = getattr(Rules, func_name)
```

Comment:
unless you add a default return to getattr it will not reach your value error.
`func = getattr(Rules, func_name, None)` should do the trick